packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.html

```diff
@@ -1,4 +1,54 @@
 <template>
-  <lightning-combobox name="sort" value={value} placeholder="Sort by" options={options} onchange={handleChange}>
```

Comment:
The `lightning-combobox` seems to be the perfect component for us except for the styling and icon (in regard to the mock-ups).

It seems to me we are going to put quite some effort to re-implementing lightning components. I wonder if we should try and use them as-is as much as we can.

Reply:
I totally agree with you and Fred and I talked about this a little bit on slack so I'd love to know what you think. An issue I found with the lighting-combobox is it is built to take a `label` value. Because of this it creates an empty element above it that's annoying. Could probably be hidden with css too. Not sure what's best. 
<img width="353" alt="Screen Shot 2021-07-19 at 2 27 00 PM" src="https://user-images.githubusercontent.com/16785453/126233190-e5a6e51f-79bc-4462-99f4-331debe1fff4.png">
<img width="973" alt="Screen Shot 2021-07-19 at 2 27 44 PM" src="https://user-images.githubusercontent.com/16785453/126233254-a7faac8c-dcef-43e8-a34c-758e84696a7a.png">

Reply:
Oh, I see. I was not aware of this extra element for the label. Not sure what's best either.

Reply:
I would personally advise to use pre-built salesforce component as much as possible.

And that the label added/required by lightning-combobox is probably there for good reasons (accessibility).

Reply:
@adroletCoveo what do you think of having a "Sort by" label on top of the combobox component?